matchzoo/losses.py

```diff
@@ -0,0 +1,56 @@
+"""Self defined loss function."""
+import typing
+from keras import backend as K
+from keras import layers
+
+# TODO: Loss function parameters setting.
+margin = 1.0
+neg_num = 1
```

Comment:
The loss parameter here? Why?

Reply:
It is concise to set loss parameter as global parameter in losses package, although it may leads to some permission problems.